packages/carbon-addons-boomerang-react/src/components/FeatureHeader/FeatureHeader.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import { settings } from 'carbon-components';
+import { default as Tabs } from '../FeatureNavTabs';
+import { default as Tab } from '../FeatureNavTab';
+import BreadcrumbNav from '../BreadcrumbNav';
+
+const { prefix } = settings;
+
+const FeatureHeader = ({
+  children,
+  className,
+  includeBorder,
+  navItems,
+  tabs,
+  tabsProps,
+  style,
+  ...rest
+}) => {
+  const classNames = classnames(`${prefix}--bmrg-feature-header`, className);
+  const additionalStyle = includeBorder ? { borderBottom: '0.0625rem solid #c1c7cd' } : {};
+
+  return (
+    <header className={classNames} style={{ ...additionalStyle, ...style }} {...rest}>
+      {navItems.length ? <BreadcrumbNav navItems={navItems} /> : null}
```

Comment:
I am thinking that this might work better if we have `slots` for the different sections of the header where the user can pass in a component vs having it all be essentially config driven. I don't see any styles that are specific to the components so that would work and give the user more control. I also like the idea of having the subtitle and title fixed so that is consistent. It could be props, or another component that is exported to give more control. It look something like

```
<FeatureHeader 
   header={<BreadcrumbNav />} 
   body={<section>content here</section>} 
   footer={<FeatureNavTabs><FeatureNavTab />...}
   title="Title"
   subtitle="Subtitle" />
```

The other option would be to just have everything be rendered by children. That would be max control but might not make our lives that much easier. I bring all of this up bc I see existing FeatureHeaders that aren't supported by this implementation. See the Launchpad team view where the "Leave Team" button is inline w/ the NavTabs. I wouldn't want to have to absolutely position a button though might not be the end of the world.